src/views/DogCard/DogDetail.css

```diff
@@ -0,0 +1,16 @@
+.test{
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center; 
+}
+
+.container{
+    background-color: rgb(170, 159, 159);
+    border: 10px solid black;
+}
+
```

Comment:
You can also target the img element here to apply a height and width so that the images are not so large on the detail page.